deluge/ui/console/cmdline/commands/move.py

```diff
@@ -46,7 +46,8 @@ def handle(self, options):
         for t_id in options.torrent_ids:
             tid = self.console.match_torrent(t_id)
             ids.extend(tid)
-            names.append(self.console.get_torrent_name(tid))
+            for name in tid:
+                names.append(self.console.get_torrent_name(name))
```

Comment:
This is a bit confusing with `for name in tid` and we could simplify this code a lot:

```suggestion
    ids = self.console.match_torrents(options.torrent_ids)
    names = [self.console.get_torrent_name(id) for id in ids]
```

Interestingly we are using match_torrent which supports any string, so it's an undocumented feature that you can move torrents by name too...